sdk/tables/aztable/connection_string_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package aztable
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConnectionStringParser(t *testing.T) {
+	require := require.New(t)
+
+	connStr := "DefaultEndpointsProtocol=https;AccountName=dummyaccount;AccountKey=secretkeykey;EndpointSuffix=core.windows.net"
```

Comment:
Do users get back a sensible error if they happened to paste the URL into 'AccountName' for some reason? Or even the full hostname (ie, accountname.core.windows.net)? Just curious what the service tells them.

Reply:
That's a good question, I would assume it will be an authentication error which from personal experience have been vague. I will make a note to investigate.